src/sort-lines.ts

```diff
@@ -27,25 +37,16 @@ function sortLines(textEditor: vscode.TextEditor, startLine: number, endLine: nu
     removeBlanks(lines);
   }
 
-  lines.sort(algorithm);
-
-  if (removeDuplicateValues) {
-    removeDuplicates(lines, algorithm);
-  }
+  lines = transformers.reduce((lines, transform) => transform(lines), lines);
 
   return textEditor.edit(editBuilder => {
     const range = new vscode.Range(startLine, 0, endLine, textEditor.document.lineAt(endLine).text.length);
     editBuilder.replace(range, lines.join('\n'));
   });
 }
 
-function removeDuplicates(lines: string[], algorithm: SortingAlgorithm | undefined): void {
-  for (let i = 1; i < lines.length; ++i) {
-    if (algorithm ? algorithm(lines[i - 1], lines[i]) === 0 : lines[i - 1] === lines[i]) {
```

Comment:
this is the only part I wasn't able to preserve: in my implementation duplicates are just equal lines, algorithm is not applied to them before this check. but none of current commands passed algorithm to this function, so hopefully that is more or less ok.